### ISSUE TITLE
Option to only allow audio by external output

### ISSUE THREAD

**Issue by MyxFoundMyOldAccount** [laurencedawson/reddit-sync-development#1963]
This would stop sound from videos being played through phone speakers, oftentimes (at least for me) an unwanted outcome.

**Comment by OvermindDL1** [comment]
Yeah having a speaker/mute icon that is optionally muted by default on videos would be perfect as well, in addition to setting the output source.  :-)
